### ISSUE TITLE
Unable to accept chef license.

### ISSUE THREAD

**Issue by hnelson** [chef/chef-workstation#370]
When I try to run Chef in Test Kitchen I get the failure shown below.  I installed Chef-Workstation from ```deb https://packages.chef.io/repos/apt/stable bionic main```. 

I've tried several methods for setting the license found on [the chef licensing page](https://docs.chef.io/chef_license_accept.html), however none of them work.
* ```chef --chef-license accept``` command: returns 'invalid option'.
* ```CHEF_LICENSE=accept kitchen converge``` Did not do anything different then without the environment variable set.
* Setting the node attribute via the ```kitchen.yml``` file didn't work either.
```
suites:
  - name: default
    run_list:
      - recipe[vpn::default]
    verifier:
      inspec_tests:
        - test/integration/default
    attributes:
      chef_client:
        chef_license: accept
```
* I also tried the ```config.rb``` setting:  ```chef_license = 'accept'``` to no avail.

In all cases, I received output similar to what is shown below.
```
hnelson@stallman:~/dev/chef/vpn$ kitchen converge
-----> Starting Kitchen (v1.24.0)
-----> Converging <default-ubuntu-1804>...
       Preparing files for transfer
       Preparing dna.json
       Resolving cookbook dependencies with Berkshelf 7.0.7...
       Removing non-cookbook files before transfer
       Preparing validation.pem
       Preparing client.rb
-----> Chef installation detected (install only if missing)
       Transferring files to <default-ubuntu-1804>
       +---------------------------------------------+
            Chef License Acceptance

       Before you can continue, 2 product licenses
       must be accepted. View the license at
       https://www.chef.io/end-user-license-agreement/

       Licenses that need accepting:
         * Chef Infra Client
         * Chef InSpec

       Do you accept the 2 product licenses (yes/no)?


       > Prompt timed out. Use non-interactive flags or enter an answer within 60 seconds.

       If you do not accept this license you will
       not be able to use Chef products.

       Do you accept the 2 product licenses (yes/no)?

       > Prompt timed out. Use non-interactive flags or enter an answer within 60 seconds.
       +---------------------------------------------+
       Chef Infra Client cannot execute without accepting the license
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Converge failed on instance <default-ubuntu-1804>.  Please see .kitchen/logs/default-ubuntu-1804.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration

hnelson@stallman:~/dev/chef/vpn$
hnelson@stallman:~/dev/chef/vpn$ chef --chef-license accept
invalid option: --chef-license
Usage:
    chef -h/--help
    chef -v/--version
    chef command [arguments...] [options...]


Available Commands:
    exec                    Runs the command in context of the embedded ruby
    env                     Prints environment variables used by ChefDK
    gem                     Runs the `gem` command in context of the embedded ruby
    generate                Generate a new app, cookbook, or component
    shell-init              Initialize your shell to use ChefDK as your primary ruby
    install                 Install cookbooks from a Policyfile and generate a locked cookbook set
    update                  Updates a Policyfile.lock.json with latest run_list and cookbooks
    push                    Push a local policy lock to a policy group on the server
    push-archive            Push a policy archive to a policy group on the server
    show-policy             Show policyfile objects on your Chef Server
    diff                    Generate an itemized diff of two Policyfile lock documents
    provision               Provision VMs and clusters via cookbook
    export                  Export a policy lock as a Chef Zero code repo
    clean-policy-revisions  Delete unused policy revisions on the server
    clean-policy-cookbooks  Delete unused policyfile cookbooks on the server
    delete-policy-group     Delete a policy group on the server
    delete-policy           Delete all revisions of a policy on the server
    undelete                Undo a delete command
    describe-cookbook       Prints cookbook checksum information used for cookbook identifier
hnelson@stallman:~/dev/chef/vpn$ chef --version
Chef Workstation: 0.2.53
  chef-run: 0.2.8
  chef-client: 14.10.9
  delivery-cli: 0.0.52 (9d07501a3b347cc687c902319d23dc32dd5fa621)
  berks: 7.0.7
  test-kitchen: 1.24.0
  inspec: 3.6.6
hnelson@stallman:~/dev/chef/vpn$
```

**Comment by tyler-ball** [comment]
Thanks for filing the issue @hnelson - the issue is that Test Kitchen master has the correct fixes in it, but we are still in the process of releasing Chef Workstation to include that change. In the meantime please [pin to Chef 14](https://docs.chef.io/chef_license_accept.html#pin-to-chef-14) until we get the releases finished

**Comment by afiune** [comment]
@hnelson I have verified that this issue has been fixed, please let us know if you have any further issues, thank you for your feedback.

Here is a full test from a brand new install, cookbook, kitchen converge:
```
➜  chef generate cookbook foo
Ignoring ffi-1.9.25 because its extensions are not built. Try: gem pristine ffi --version 1.9.25
Generating cookbook foo
- Ensuring correct cookbook content
- Committing cookbook files to git

Your cookbook is ready. Type `cd foo` to enter it.

There are several commands you can run to get started locally developing and testing your cookbook.
Type `delivery local --help` to see a full list of local testing commands.

Why not start by writing an InSpec test? Tests for the default recipe are stored at:

test/integration/default/default_test.rb

If you'd prefer to dive right in, the default recipe can be found at:

recipes/default.rb

➜  cd foo
➜  foo git:(master) kitchen converge default-centos-7
Ignoring ffi-1.9.25 because its extensions are not built. Try: gem pristine ffi --version 1.9.25
-----> Starting Kitchen (v2.3.2)
-----> Creating <default-centos-7>...
       ==> vagrant: A new version of Vagrant is available: 2.2.5 (installed version: 2.1.5)!
       ==> vagrant: To upgrade visit: https://www.vagrantup.com/downloads.html

       Bringing machine 'default' up with 'virtualbox' provider...
       ==> default: Box 'bento/centos-7' could not be found. Attempting to find and install...
           default: Box Provider: virtualbox
           default: Box Version: >= 0
       ==> default: Loading metadata for box 'bento/centos-7'
           default: URL: https://vagrantcloud.com/bento/centos-7
       ==> default: Adding box 'bento/centos-7' (v201907.24.0) for provider: virtualbox
           default: Downloading: https://vagrantcloud.com/bento/boxes/centos-7/versions/201907.24.0/providers/virtualbox.box
==> default: Successfully added box 'bento/centos-7' (v201907.24.0) for 'virtualbox'!
       ==> default: Importing base box 'bento/centos-7'...
==> default: Matching MAC address for NAT networking...
       ==> default: Checking if box 'bento/centos-7' is up to date...
       ==> default: Setting the name of the VM: kitchen-foo-default-centos-7-1662301a-01de-48c5-b8ff-3d2d5e92b4ce
       ==> default: Clearing any previously set network interfaces...
       ==> default: Preparing network interfaces based on configuration...
           default: Adapter 1: nat
       ==> default: Forwarding ports...
           default: 22 (guest) => 2222 (host) (adapter 1)
       ==> default: Running 'pre-boot' VM customizations...
       ==> default: Booting VM...
       ==> default: Waiting for machine to boot. This may take a few minutes...
           default: SSH address: 127.0.0.1:2222
           default: SSH username: vagrant
           default: SSH auth method: private key
           default:
           default: Vagrant insecure key detected. Vagrant will automatically replace
           default: this with a newly generated keypair for better security.
           default:
           default: Inserting generated public key within guest...
           default: Removing insecure key from the guest if it's present...
           default: Key inserted! Disconnecting and reconnecting using new SSH key...
       ==> default: Machine booted and ready!
       ==> default: Checking for guest additions in VM...
           default: The guest additions on this VM do not match the installed version of
           default: VirtualBox! In most cases this is fine, but in rare cases it can
           default: prevent things such as shared folders from working properly. If you see
           default: shared folder errors, please make sure the guest additions within the
           default: virtual machine match the version of VirtualBox you have installed on
           default: your host and reload your VM.
           default:
           default: Guest Additions Version: 6.0.10
           default: VirtualBox Version: 5.2
       ==> default: Setting hostname...
       ==> default: Mounting shared folders...
           default: /tmp/omnibus/cache => /Users/afiune/.kitchen/cache
       ==> default: Machine not provisioned because `--no-provision` is specified.
       [SSH] Established
       Vagrant instance <default-centos-7> created.
       Finished creating <default-centos-7> (1m12.42s).
-----> Converging <default-centos-7>...
       Preparing files for transfer
       Policy lock file doesn't exist, running `chef install` for Policyfile /Users/afiune/chef/workspace/foo/Policyfile.rb...
       Ignoring ffi-1.9.25 because its extensions are not built. Try: gem pristine ffi --version 1.9.25
       Building policy foo
       Expanded run list: recipe[foo::default]
       Caching Cookbooks...
       Installing foo >= 0.0.0 from path

       Lockfile written to /Users/afiune/chef/workspace/foo/Policyfile.lock.json
       Policy revision id: aca7965e2c65a8a2f9615c2bb347683326f3ff56de4af7048b6520c1cb44f05c
       Preparing dna.json
       Exporting cookbook dependencies from Policyfile /var/folders/wl/2vw3s05d3z5d7pj7dgkzr0fc0000gn/T/default-centos-7-sandbox-20190910-20271-9j436m...
       Ignoring ffi-1.9.25 because its extensions are not built. Try: gem pristine ffi --version 1.9.25
       Exported policy 'foo' to /var/folders/wl/2vw3s05d3z5d7pj7dgkzr0fc0000gn/T/default-centos-7-sandbox-20190910-20271-9j436m

       To converge this system with the exported policy, run:
         cd /var/folders/wl/2vw3s05d3z5d7pj7dgkzr0fc0000gn/T/default-centos-7-sandbox-20190910-20271-9j436m
         chef-client -z
       Removing non-cookbook files before transfer
       Preparing validation.pem
       Preparing client.rb
-----> Installing Chef install only if missing package
       Downloading https://omnitruck.chef.io/install.sh to file /tmp/install.sh
       Trying wget...
       Download complete.
       el 7 x86_64
       Getting information for chef stable  for el...
       downloading https://omnitruck.chef.io/stable/chef/metadata?v=&p=el&pv=7&m=x86_64
         to file /tmp/install.sh.3293/metadata.txt
       trying wget...
       sha1	ce8e362fcc5d98302bd977919b23eb30acd5f3cb
       sha256	56e3267ee1c2f0aab2a85ac49ae24a328a235889ce037c8ad6ade996d38bc849
       url	https://packages.chef.io/files/stable/chef/15.2.20/el/7/chef-15.2.20-1.el7.x86_64.rpm
       version	15.2.20
       downloaded metadata file looks valid...
       downloading https://packages.chef.io/files/stable/chef/15.2.20/el/7/chef-15.2.20-1.el7.x86_64.rpm
         to file /tmp/omnibus/cache/chef-15.2.20-1.el7.x86_64.rpm
       trying wget...
       Comparing checksum with sha256sum...

       WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING

       You are installing an omnibus package without a version pin.  If you are installing
       on production servers via an automated process this is DANGEROUS and you will
       be upgraded without warning on new releases, even to new major releases.
       Letting the version float is only appropriate in desktop, test, development or
       CI/CD environments.

       WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING

       Installing chef
       installing with rpm...
       warning: /tmp/omnibus/cache/chef-15.2.20-1.el7.x86_64.rpm: Header V4 DSA/SHA1 Signature, key ID 83ef826a: NOKEY
       Preparing...                          ################################# [100%]
       Updating / installing...
          1:chef-15.2.20-1.el7               ################################# [100%]
       Thank you for installing Chef Infra Client! For help getting started visit https://learn.chef.io
       Transferring files to <default-centos-7>
       +---------------------------------------------+
       ✔ 2 product licenses accepted.
       +---------------------------------------------+
       Starting Chef Infra Client, version 15.2.20
       Creating a new client identity for default-centos-7 using the validator key.
       Using policy 'foo' at revision 'aca7965e2c65a8a2f9615c2bb347683326f3ff56de4af7048b6520c1cb44f05c'
       resolving cookbooks for run list: ["foo::default@0.1.0 (8f495e7)"]
       Synchronizing Cookbooks:
         - foo (0.1.0)
       Installing Cookbook Gems:
       Compiling Cookbooks...
       Converging 0 resources

       Running handlers:
       Running handlers complete
       Chef Infra Client finished, 0/0 resources updated in 01 seconds
       Downloading files from <default-centos-7>
       Finished converging <default-centos-7> (0m18.60s).
-----> Kitchen is finished. (1m33.86s)
➜  foo git:(master) kitchen destroy
```
